xls/modules/fpsub_2x64.x

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file implements most of IEEE-754 single-precision
+// floating-point subtraction, with the following exceptions:
```

Comment:
Moved to apfloat file since comparisons no longer depend on subtraction (using exp + sfd comparison).

RE grouping all calls into 32/64/etc file - I like this idea a lot from a usability standpoint.  However, I think the main reason we have most of these floating point units as 1 module per file is so that we can do JIT wrapper testing (JIT wrapper wraps entry function of a .x file).  Not sure of a great way around this.